ops/go.work

```diff
@@ -0,0 +1,7 @@
+go 1.23.1
+
+use (
+	.
+	../validation
+)
```

Comment:
Not sure what the purpose of this file is, haven't seen it used in this way before. We did have one previously but it was in the root of the repo. Can you clarify?

Reply:
`ops` needs it so that local changes in the validation package carry through without requiring manual version bumps.